test/e2e/specs/editor/various/adding-patterns.spec.js

```diff
@@ -14,15 +14,13 @@ test.describe( 'adding patterns', () => {
 		await page.getByRole( 'tab', { name: 'Patterns' } ).click();
 		await page.fill(
 			'role=region[name="Block Library"i] >> role=searchbox[name="Search for blocks and patterns"i]',
-			'Social links with a shared background color'
+			'Standard'
 		);
 
-		await page.click(
-			'role=option[name="Social links with a shared background color"i]'
-		);
+		await page.click( 'role=option[name="Standard"i]' );
```

Comment:
Nit, non-blocking: We now prefer `getByRole` wherever possible.

```suggestion
		await page.getByRole( 'option', { name: 'Standard' } ).click();
```